src/cli/utils/sharepoint.go

```diff
@@ -105,6 +108,20 @@ func ValidateSharePointRestoreFlags(backupID string, opts SharePointOpts) error
 		}
 	}
 
+	if _, ok := opts.Populated[flags.ListTemplateFN]; ok {
+		found := false
+
+		for _, wu := range opts.ListTemplate {
+			if wu == flags.InvalidListTemplate {
+				found = true
+			}
+		}
```

Comment:
you can use the slices package here.  `slices.Contains(opts.ListTemplate, flags.InvalidListTemplate)`